src/EnergyPlus/ChillerExhaustAbsorption.cc

```diff
@@ -131,7 +131,14 @@ namespace EnergyPlus::ChillerExhaustAbsorption {
 
     void ExhaustAbsorberSpecs::simulate(EnergyPlusData &state, const PlantLocation &calledFromLocation, bool FirstHVACIteration, Real64 &CurLoad, bool RunFlag)
     {
-        int brIdentity(0);
+        enum brLoopType
+        {
+            CHILLER,
+            HEATER,
+            CONDENSER,
+            NOMATCH
+        };
+        brLoopType brIdentity(NOMATCH);
```

Comment:
This is much closer, but not quite there.  I'm confused why this enum is repeated in the two cases.  It should be declared once.  Please:
- move this one enum declaration to either the top of the ChillerExhaustAbsorption.cc file, or in the header,
- make it an `enum class` instead of a raw enum,
- change the enum name from camelCase to PascalCase,
- change the member names from `SCREAMINGCASE to `PascalCase`,
- and make the accompanying change for the `enum class` by using the namespace below (`BrLoopType::Chiller`)

Reply:
> 
> 
> This is much closer, but not quite there. I'm confused why this enum is repeated in the two cases. It should be declared once. Please:
> 
>     * move this one enum declaration to either the top of the ChillerExhaustAbsorption.cc file, or in the header,
> 
>     * make it an `enum class` instead of a raw `enum`,
> 
>     * change the enum name from `camelCase` to `PascalCase`,
> 
>     * change the member names from `SCREAMINGCASE` to `PascalCase`,
> 
>     * and make the accompanying change for the `enum class` by using the namespace below (`BrLoopType::Chiller`)

I will make the changes. For why it was done twice in the two function: actually these variable is very "local" to the getDesignCapacities() function so I did not want to bump up the scope. 
It might be a little bit debatable since it is just a type declaration. Or we can make it an even higher level scope, to that both the ChillerExhaustAbsorption.cc (or hh) and the ChillerGasAbsorption.cc (or .hh). 

Any comments or thoughts?

Reply:
Wherever it makes sense is fine with me.  But duplicating the same list of enumarations in two places smells like it needs to be DRYd up.  For now, I say just move it up to the file level here so it can be shared between these two functions without duplication.  

And you are correct about the type declaration comment, the scope here doesn't matter since an enum is just optimized away quickly, so it won't affect runtime.  The scope should simply be determined by how many components this can be applied to.

Reply:
So putting the definition in EnergyPlus/Plant/Enum.hh seems to be a good choice; then add `#inlcude<EnergyPlus/Plant/Enum.hh>` to both ChillerExhaustAbsorption.cc and ChillerGasAbsorption.cc.

Reply:
Or definition in EnergyPlus/Plant/DataPlant.hh, which is already included in both both ChillerExhaustAbsorption.cc and ChillerGasAbsorption.cc.

Reply:
See the new commits.